content/director-blobstore-signed-urls.md

```diff
@@ -20,11 +20,15 @@ following properties:
 * `blobstore.enable_signed_urls`: Set this to true to have the director begin
   sending signed urls to the agent.
 
-You must continue configuring `blobstore.*` properties. Enabling signed URLs
-should work alongside blobstore provider specific encryption options such as
-`blobstore.encryption_key` (GCS) and `blobstore.sse_kms_key_id` (AWS).
-
-An ops-file in bosh-deployment can be used to enable signed urls. See: https://github.com/cloudfoundry/bosh-deployment/blob/master/enable-signed-urls.yml
+You must continue configuring `blobstore.*` properties, except
```

Comment:
@bgandon I would propose following. What is your feedback on that?

Enabling signed URLs should work alongside blobstore provider
specific encryption options such as `blobstore.encryption_key` (GCS) and
`blobstore.sse_kms_key_id` (AWS). You must continue configuring `blobstore.*` properties. For
`blobstore.agent.user` and `blobstore.agent.password` you can configure dummy values because they are no more used but still required in the templates of some CPIs.  
An ops-file in bosh-deployment can be used to enable signed URLs and manage the unneccesary properites.
See the [bosh-deployment › misc/blobstore-signed-urls.yml](https://github.com/cloudfoundry/bosh-deployment/blob/master/misc/blobstore-signed-urls.yml)
ops file.

Reply:
Makes sense, I've adopted that text.